build.cmd

```diff
@@ -132,6 +133,17 @@ if /i not "%1" == "msbuildargs" goto SkipMsbuildArgs
 :CollectMsbuildArgs
 shift
 if "%1"=="" goto ArgsDone
+if /i [%1] == [/p:OfficialBuildId] (
```

Comment:
As we chatted we want to do this as a separate step from the normal build flow so please make sure we track that work item.